src/explore-education-statistics-admin/src/pages/release/data/utils/getUnmappedFilterErrors.ts

```diff
@@ -0,0 +1,35 @@
+import { MappableFilter } from '@admin/pages/release/data/utils/getApiDataSetFilterMappings';
+import { FiltersMapping } from '@admin/services/apiDataSetVersionService';
+import { ErrorSummaryMessage } from '@common/components/ErrorSummary';
+import typedKeys from '@common/utils/object/typedKeys';
+import { Dictionary } from '@common/types';
+
+export default function getUnmappedFilterErrors(
+  mappableFilters: Dictionary<MappableFilter[]>,
+  filtersMapping: FiltersMapping,
+): ErrorSummaryMessage[] {
+  const errors: ErrorSummaryMessage[] = [];
+
+  typedKeys(mappableFilters).forEach(filterKey => {
+    // For MVP you can't map columns or options in unmapped columns
+    if (!filtersMapping.candidates[filterKey]) {
+      return;
+    }
+
+    const total = mappableFilters[filterKey]?.filter(
+      map => map.mapping.type === 'AutoNone',
+    ).length;
```

Comment:
This is inefficient as we're allocating an array just to count its length. Would use `sumBy` instead.